apps/prairielearn/src/sprocs/authz_assessment.sql

```diff
@@ -16,11 +16,13 @@ CREATE FUNCTION
         OUT show_closed_assessment_score boolean, -- If students can view their grade after the assessment is closed
         OUT active boolean,         -- If the assessment is visible but not active
         OUT next_active_time text,  -- The next time the assessment becomes active. This is non-null only if the assessment is not currently active but will be later.
-        OUT access_rules JSONB       -- For display to the user. The currently active rule is marked by 'active' = TRUE.
+        OUT access_rules JSONB,       -- For display to the user. The currently active rule is marked by 'active' = TRUE.
+        OUT group_id INT
```

Comment:
When is this value ever used by a caller of this sproc?

Reply:
fixed in 847e59d22cda7128a1c139c9769ccddb52bf3f24